openedx/core/djangoapps/content/course_overviews/migrations/0007_auto__add_field_courseoverview_tabs.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'CourseOverview.tabs'
+        db.add_column('course_overviews_courseoverview', 'tabs',
```

Comment:
Rather than storing the tabs information as a string, why not create a separate SQL table for CourseTabs?  Right now, you are serializing only the tab's `tab.id`.  In the future, we'll want to extend this to include it's name, is_hidden field, etc.